definitions.go

```diff
@@ -757,3 +757,45 @@ type RecoveryCodeRepresentation struct {
 type ActivationCodeRepresentation struct {
 	Code *string `json:"code,omitempty"`
 }
+
+// CPREmailRepresentation struct
+type CPREmailRepresentation struct {
+	Recipient   *string                         `json:"recipient,omitempty"`
+	Theming     *CPREmailThemingRepresentation  `json:"theming,omitempty"`
+	Attachments *[]CPRAttachementRepresentation `json:"attachments,omitempty"`
+}
+
+// CPREmailThemingRepresentation struct
+type CPREmailThemingRepresentation struct {
+	SubjectKey         *string                              `json:"subjectKey,omitempty"`
+	SubjectParameters  *[]string                            `json:"subjectParameters,omitempty"`
+	Template           *string                              `json:"template,omitempty"`
+	TemplateParameters *CPRTemplateParametersRepresentation `json:"templateParameters,omitempty"`
```

Comment:
Remove this TemplateParametersRepresentation type and simply use a map[string]string